packages/core/src/modules/proofs/index.ts

```diff
@@ -12,3 +12,6 @@ export * from './ProofsApiOptions'
 // Module
 export * from './ProofsModule'
 export * from './ProofsModuleConfig'
+
+// Services
+export * from './services'
```

Comment:
I was thinking we create a separate module for this already in AFJ, as the current proofs module is mainly focused on didcomm, and I think it's good to keep that separated from functionality that is not related to didcomm.

Reply:
Yeah, happy to do it as a module already.

Reply:
Would that just be a module without a public API then?

Reply:
It is a new module now that just contains the service, no public API.